### DISCUSSION TITLE
Should we rename this repo now? Before we get too many references or attached to RIPT?

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/QRiS#25] [Q&A] [answered]
As we've discussed, I like the RIPT acronym, but its our same old problem of new names. I think we should move on to calling this thing "Riverscapes Studio - Desktop QGIS" and maybe name the repo something horrible like RS_DQGIS and eventually [create a subdomain for documentation site](https://riverscapes.xyz/Tools/Technical_Reference/Documentation_Standards/WebSites/subdomain.html) of something like studio.riverscapes.xyz and that will also serve the web desktop and field studio versions).  I seriously doubt that RIPT would survive a review by our information architect. 

Thoughts @MattReimer, @philipbaileynar and @nick4rivers?

**Comment by nick4rivers** [comment, marked as answer]
@joewheaton - I think we are all liking the "Riverscapes Studio" handle better than RIPT. I'll leave it up to @MattReimer and @philipbaileynar to make decisions on whether we can wait to commit to a name change? or if indeed it's better to just do this immediately.

Otherwise - I @joewheaton 's idea for include "QGIS" in the name for clarity. So something his suggestions, or even "QGIS Riverscapes Studio - QRiS"?

**Reply by joewheaton** [reply]
I kind of like your **QRiS** (it sounds like curious) so what would regular vanilla one be **RiS**?

**Reply by MattReimer** [reply]
We can name it however you want. I don't get a say though 😅 

Sooner is definitely better than later.

**Reply by joewheaton** [reply]
Cool @MattReimer . @philipbaileynar your 2 cents CAD?

**Reply by philipbaileynar** [reply]
I'm fine with QRIS. I prefer keeping Riverscapes Studio for the streamline, more focused, online experience. 

My only hard and fast rule is that whatever name we choose, is locked in for 12 months. Renaming stuff constantly is a waste of time.

**Reply by nick4rivers** [reply]
Let's roll with "Riverscapes Studio"
  - my only concern is it suggests a lot of capability / functionality - hope we can live up to the hype.
  - QRIS it is....

**Reply by MattReimer** [reply]
This has been done. Behold:

https://github.com/Riverscapes/QRiS
